ecom-mockman/src/Routes/PageRoutes.jsx

```diff
@@ -2,17 +2,17 @@ import React from 'react'
 import Mockman from "mockman-js";
 import {Routes,Route} from "react-router-dom"
 import Home from '../pages/Home/Home'
-import Login from "../pages/Auth/login"
-import Products from "../pages/Products/Products"
-import Cart from "../pages/Cart/cart"
+
+import Products from  "../pages/Products/Products"
```

Comment:
you can use barrel export

Reply:
okay let me see, thanks for the suggestion